azure-commons-plugin/pom.xml

```diff
@@ -15,6 +15,12 @@
     <description>Common APIs for Azure related Jenkins plugins</description>
     <url>https://wiki.jenkins-ci.org/display/JENKINS/Azure+Commons+Plugin</url>
 
+    <properties>
+        <enableAzureApiTrace>false</enableAzureApiTrace>
```

Comment:
what if azure api call failed? will there be trace?

Reply:
I can still trace the failed api call if necessary. Our BI does not track these data for now.